test/functional/wallet_avoidreuse.py

```diff
@@ -138,51 +145,52 @@ def test_fund_send_fund_senddirty(self):
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 1 single, unused 10 btc output
+        # listunspent should show 1 single, unused 10 btc output.
         assert_unspent(self.nodes[1], total_count=1, total_sum=10, reused_supported=True, reused_count=0)
-        # getbalances should show no used, 10 btc trusted
+        # getbalances should show no used, 10 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 0, "trusted": 10})
-        # node 0 should not show a used entry, as it does not enable avoid_reuse
+        # Node 0 should not show a used entry, as it does not enable avoid_reuse.
         assert("used" not in self.nodes[0].getbalances()["mine"])
 
         self.nodes[1].sendtoaddress(retaddr, 5)
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 1 single, unused 5 btc output
+        # listunspent should show 1 single, unused 5 btc output.
         assert_unspent(self.nodes[1], total_count=1, total_sum=5, reused_supported=True, reused_count=0)
-        # getbalances should show no used, 5 btc trusted
+        # getbalances should show no used, 5 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 0, "trusted": 5})
 
         self.nodes[0].sendtoaddress(fundaddr, 10)
         self.nodes[0].generate(1)
         self.sync_all()
 
-        # listunspent should show 2 total outputs (5, 10 btc), one unused (5), one reused (10)
+        # listunspent should show 2 total outputs (5, 10 btc), one unused (5), one reused (10).
         assert_unspent(self.nodes[1], total_count=2, total_sum=15, reused_count=1, reused_sum=10)
-        # getbalances should show 10 used, 5 btc trusted
+        # getbalances should show 10 used, 5 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 10, "trusted": 5})
 
         self.nodes[1].sendtoaddress(address=retaddr, amount=10, avoid_reuse=False)
 
-        # listunspent should show 1 total outputs (5 btc), unused
+        # listunspent should show 1 total outputs (5 btc), unused.
         assert_unspent(self.nodes[1], total_count=1, total_sum=5, reused_count=0)
-        # getbalances should show no used, 5 btc trusted
+        # getbalances should show no used, 5 btc trusted.
         assert_balances(self.nodes[1], mine={"used": 0, "trusted": 5})
 
-        # node 1 should now have about 5 btc left (for both cases)
+        # Node 1 should now have about 5 btc left (for both cases).
         assert_approx(self.nodes[1].getbalance(), 5, 0.001)
         assert_approx(self.nodes[1].getbalance(avoid_reuse=False), 5, 0.001)
 
     def test_fund_send_fund_send(self):
-        '''
+        """
         Test the simple case where [1] generates a new address A, then
-        [0] sends 10 BTC to A.
-        [1] spends 5 BTC from A. (leaving roughly 5 BTC useable)
-        [0] sends 10 BTC to A again.
-        [1] tries to spend 10 BTC (fails; dirty).
+        [0] sends 10 BTC to A
```

Comment:
Here you remove the dots, elsewhere you add them?

Reply:
This is a list of partial sentences.